### ISSUE TITLE
Das Erste HD seems to be broken again

### ISSUE THREAD

**Issue by rdorsch** [jnk22/kodinerds-iptv#234]
tvheadend stops with "An unknown error occured". Other channels are working...

**Comment by gsantner** [comment]
Didn't change this one for quite some time (also not with recent commits), works fine for me . <https://daserstehdde-lh.akamaihd.net/i/daserstehd_de@629196/index_7_av-p.m3u8>

**Comment by rdorsch** [comment]
Works again here. I had to  map the service again. Initially not even that was possible, but maybe a tvheadend restart fixed that again.

I had to map the service multiple times now for Das Erste HD, I see that very rarely for other channels....

I think we can close the issue again (except if somebody know how to fix the remapping issue, though this could be an issue with tvheadend...).

Thanks
Rainer
